modules/docs/utils/build-docs.js

```diff
@@ -51,3 +52,77 @@ const contents = fs
   .toString()
   .replace('/* DOCS_REPLACED_BY_BUILD */', docsStr.substring(1, docsStr.length - 1)); // remove the square bracket on the ends
 fs.writeFileSync(docsFilePath, contents);
+
+/**
+ * The following ensures that the stackblitz files are copied to the dist folder and that the current version is included.
+ */
+// Source and destination base paths
+const sourceBase = path.join(__dirname, '../lib/stackblitzFiles');
+const destBase = path.join(__dirname, '../dist/es6/lib/stackblitzFiles');
+const rootPackageJsonPath = path.join(__dirname, '..', 'package.json');
+const searchPath = path.join(__dirname, '../dist/es6/lib/stackblitzFiles/packageJSONFile.ts');
+const rootPackageJson = JSON.parse(fs.readFileSync(rootPackageJsonPath, 'utf8'));
+const version = rootPackageJson.version || '0.0.0';
+
+// Ensure the destination directory exists
+if (!fs.existsSync(destBase)) {
+  fs.mkdirSync(destBase, {recursive: true});
+}
+
+glob(`${sourceBase}/**`, {dot: true}, (err, files) => {
+  if (err) {
+    console.error('Error finding files:', err);
+    return;
+  }
+
+  if (!files || files.length === 0) {
+    console.log('No files found in:', sourceBase);
+    return;
+  }
+
+  files
+    .filter(file => !fs.lstatSync(file).isDirectory()) // Filter out directories
+    .filter(
+      file => !file.includes(`${path.sep}types${path.sep}`) && !file.endsWith(`${path.sep}types`)
+    ) // Ignore 'types' subfolder
+    .forEach(file => {
+      const relativePath = path.relative(sourceBase, file); // Preserve structure
+      const destPath = path.join(destBase, relativePath); // Full destination file path
+
+      // Optional: Read the file content (if you still want it for logging or other purposes)
+      const content = fs.readFileSync(file).toString();
+      // console.log(`Content of ${file}:`, content); // Example usage
+
+      // Copy the file to the destination
+      fs.copyFile(file, destPath, err => {
+        if (err) {
+          console.error(`Error copying ${file} to ${destPath}:`, err);
+        } else {
+          console.log(`Copied ${file} to ${destPath}`);
+        }
+      });
+    });
+
+  glob(searchPath, (err, files) => {
```

Comment:
This second glob handles rewriting the `packageJSONFile.ts` to insert the correct version number.